src/browser/api/browser_view.ts

```diff
@@ -35,31 +38,44 @@ export async function create(options: BrowserViewOpts) {
         setBounds(ofView, options.bounds);
     }
 }
-
+export function hide(ofView: OfView) {
+    const win = getElectronBrowserWindow(ofView.target);
+    win.removeBrowserView(ofView.view);
+}
+export function show(ofView: OfView) {
```

Comment:
Not a fan of this either. Render process could simply add the browserview, not too much value from this abstraction.